README.md

```diff
@@ -23,7 +23,7 @@ Zowe&trade; Explorer extension modernizes the way developers and system administ
 
 Ensure that you meet the following prerequisites before using the extension:
 
-* Access to z/OSMF.
+* Configured [z/OSMF](https://docs.zowe.org/stable/user-guide/systemrequirements-zosmf.html#z-os-requirements).
 * At least, one Zowe CLI `zosmf` profile.
```

Comment:
I think it'd be good to remove the `At least, on Zowe CLI 'zosmf' profile` since we've successfully decoupled them : )